WTWSMS/history/characters/kurdish.txt

```diff
@@ -3,7 +3,7 @@
 	# Vonones II, King of Parthia in 51, King of Atropatene in 11 - 50
 	dynasty=100685
 	religion="zoroastrian"
-	culture="parthian"
+	culture="kurdish"
```

Comment:
Was he a Kurd?

Reply:
> 
> 
> Was he a Kurd?

I'm used ''replace all'' option and accidentaly changed his culture.